alpha-lab/imu-transformations/index.md

```diff
@@ -57,11 +53,11 @@ def transform_imu_to_world(imu_coordinates, imu_quaternions):
         ])
 ```
 
-Now that we have the `transform_imu_to_world` function, let's use it!
+### Example: Heading Vectors in World Coordinates
```

Comment:
I changed all the applications of the methods we defined to examples with a ### header. This way in the ## sections we introduce the transformation function and in the ### sections we show how to use it.

Reply:
@marc-tonsen Thanks!